src/mentors.json

```diff
@@ -6427,5 +6426,37 @@
         "id": "ShiftAhoy"
       }
     ]
+  },
+  {
+    "id": "daniel@dslemay.com",
+    "name": "Daniel Lemay",
+    "avatar": "https://www.dslemay.com/avatar.jpg",
+    "title": "Frontend Developer",
+    "description": "Self-taught dev. Focused on JavaScript, React, and Accessibility.",
+    "country": "US",
+    "spokenLanguages": [
+      "en"
+    ],
+    "tags": [
+      "accessibility",
+      "reactjs",
+      "career",
+      "tooling",
+      "javascript"
+    ],
+    "channels": [
+      {
+        "type": "email",
+        "id": "daniel@dslemay.com"
+      },
+      {
+        "type": "slack",
+        "id": "Daniel Lemay"
```

Comment:
SlackId is invalid.
[How to find it?](https://medium.com/@moshfeu/how-to-find-my-member-id-in-slack-workspace-d4bba942e38c#88b8)

Reply:
Thank you. Missed finding that correctly. Updated and rebased off master to resolve merge conflict.